### PR TITLE
DIG-5321: Preferred Chosen Name Confirmation Screen

### PR DESCRIPTION
**Pull request by xuhongkang**

### CODE REVIEW

File: services-js/access-boston/src/client/preferred-chosen-name/views/Index.stories.tsx
Comment (xuhongkang) [inline services-js/access-boston/src/client/preferred-chosen-name/views/Index.stories.tsx:1]: Updating Story Configs

File: services-js/access-boston/src/client/preferred-chosen-name/components/RowColumns.tsx
Comment (xuhongkang) [inline services-js/access-boston/src/client/preferred-chosen-name/components/RowColumns.tsx:1]: Removing Faulty Code

File: services-js/access-boston/src/client/preferred-chosen-name/views/confirmationView.tsx
Comment (xuhongkang) [inline services-js/access-boston/src/client/preferred-chosen-name/views/confirmationView.tsx:1]: Main Implementation

**Review by xuhongkang — COMMENTED** [review]
LGTM

**Review by phillipbentonkelly — APPROVED** [review]
LGTM

**Pull request merged**